components/badge/badge.css

```diff
@@ -0,0 +1,49 @@
+.badge {
+  position: relative;
+  width: fit-content;
+  height: fit-content;
+}
+
+.badge-heading {
+  position: absolute;
+  top: -0.5rem;
```

Comment:
Can use px instead of rem here? as anything below 1rem is not advisable to change with root font size. Correct me if I am wrong

Reply:
yes , i can use px here as i am using px for anything below 10 px as a standard .Thanks.